addons/zylann.editor_debugger/dock.gd

```diff
@@ -264,10 +258,10 @@ func _on_Tree_nothing_selected() -> void:
 
 
 func _input(event: InputEvent) -> void:
-	if event is InputEventKey:
-		if event.pressed:
-			if event.keycode == KEY_F12:
-				pick(get_viewport().get_mouse_position())
+	var event_key := event as InputEventKey
+	if event_key and event_key.pressed:
```

Comment:
I would prefer `!= null` instead of any kind of "booleanization" that isn't performed on a boolean. Also it is quite unfortunate that this change would be required to silence some warnings because it needs an extra variable for every subclass that would need to be checked in the same scope.

Reply:
I agree that this is unfortunate. Ideally GDScript should be smart enough to realize referring to `event` as though it is the subclass `EventInputKey` only after checking `if event is EventInputKey` is, in fact, type-safe. Sadly it's not that smart yet, and either it has to be done like this or with an `as EventInputKey` repeated at every place where the `event` var is used.